packages/office-ui-fabric-react/src/components/GroupedList/GroupFooter.styles.ts

```diff
@@ -17,7 +17,7 @@ export const getStyles = (props: IGroupFooterStyleProps): IGroupFooterStyles =>
         position: 'relative',
         padding: '5px 38px',
         selectors: {
-          [`:global(.${classNames.link}`]: {
+          [`& :global(.${classNames.link}`]: {
```

Comment:
Are the parentheses required here? If so, I think you're missing the closing parentheses.

Reply:
wow it's totally broken from the outset!